### PR TITLE
paperless-ngx: 1.16.1 -> 1.16.3, python3Packages.tika-client: 0.1.0 -> 0.1.1

### PR DESCRIPTION
**Pull request by leona-ya**
###### Description of changes

https://github.com/paperless-ngx/paperless-ngx/releases/tag/v1.16.3

https://github.com/stumpylog/tika-client/blob/main/CHANGELOG.md

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by e1mo — COMMENTED** [review]
How about also running the new frontend tests added in 1.16.2?

The following in fronted works for me:
```nix
    doCheck = true;
    checkPhase = ''
      npm run test
    '';

**Review by e1mo — APPROVED** [review]
LGTM, thanks!

File: pkgs/applications/office/paperless-ngx/default.nix
Comment (emilylange) [inline pkgs/applications/office/paperless-ngx/default.nix:1]: Would you mind limiting the platforms via
```patch
diff --git a/pkgs/applications/office/paperless-ngx/default.nix b/pkgs/applications/office/paperless-ngx/default.nix
index 4c3f861b2bd..1993a2895e3 100644
--- a/pkgs/applications/office/paperless-ngx/default.nix
+++ b/pkgs/applications/office/paperless-ngx/default.nix
@@ -288,6 +288,7 @@ python.pkgs.buildPythonApplication rec {
     homepage = "https://docs.paperless-ngx.com/";
     changelog = "https://github.com/paperless-ngx/paperless-ngx/releases/tag/v${version}";
     license = licenses.gpl3Only;
+    platforms = platforms.linux;
     maintainers = with maintainers; [ lukegb gador erikarvstedt ];
   };
 }
```

Preferably in an additional commit :)

Nobody picked up on the build issues on `darwin` in the meantime, and upstream does not seem to officially support `darwin` anyway.
Previously: https://github.com/NixOS/nixpkgs/pull/229923#discussion_r1186040499

**Review by emilylange — CHANGES_REQUESTED** [review]

**Review by e1mo — APPROVED** [review]

**Review by emilylange — APPROVED** [review]
Result of `nixpkgs-review pr 239239` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>paperless-ngx</li>
    <li>python310Packages.tika-client</li>
    <li>python310Packages.tika-client.dist</li>
    <li>python311Packages.tika-client</li>
    <li>python311Packages.tika-client.dist</li>
  </ul>
</details>

**Pull request merged**